v2/tools/generator/internal/codegen/pipeline/state.go

```diff
@@ -30,6 +29,14 @@ type State struct {
  *  that identifier.
  */
 
+type StateInfo string
```

Comment:
We could call this a `StateItem` or `StateElement` if we wanted?

Not sure if that's necessarily a lot clearer than `StateInfo` but might be _slightly_ less generic?

Reply:
Renamed to `StateDataKey`, along with adjusting the names of the associated functions.